airbyte-ci/connectors/qa-engine/qa_engine/models.py

```diff
@@ -14,12 +14,10 @@ class ConnectorTypeEnum(str, Enum):
     source = "source"
     destination = "destination"
 
-
-class ReleaseStageEnum(str, Enum):
+class SupportLevelEnum(str, Enum):
```

Comment:
Ahah the enum exists here :) Might be worth migrating it to connector_ops.utils...